.github/workflows/test-workflow.yml

```diff
@@ -1,6 +1,6 @@
 name: Test Workflow
 # This workflow is triggered on pull requests to master or a opendistro release branch
-on:
+on: repository_dispatch
```

Comment:
I think this can be removed?

Reply:
Yeah I think so since we are not using curl command now